forge-gui/res/cardsfolder/c/counterflux.txt

```diff
@@ -2,6 +2,6 @@ Name:Counterflux
 ManaCost:U U R
 Types:Instant
 K:This spell can't be countered.
-A:SP$ Counter | Cost$ U U R | TargetType$ Spell | TgtPrompt$ Select target spell you don't control. | ValidTgts$ Card.YouDontCtrl | SpellDescription$ Counter target spell you don't control.
-A:SP$ Counter | Cost$ 1 U U R | AllValid$ Spell.YouDontCtrl | PrecostDesc$ Overload | CostDesc$ {1}{U}{U}{R} | NonBasicSpell$ True | SpellDescription$ (You may cast this spell for its overload cost. If you do, change its text by replacing all instances of "target" with "each.") | StackDescription$ Counter each spell you don't control.
+A:SP$ Counter | TargetType$ Spell | TgtPrompt$ Select target spell you don't control. | ValidTgts$ Spell.YouDontCtrl | SpellDescription$ Counter target spell you don't control.
```

Comment:
Is this ValidTgts$ OK now? I thought it still needed to be Card?

Reply:
I need to test it again, but it should still work

Reply:
i made all Cards on the Stack be Spells

i wanted this for another MR later too

Reply:
reverted because it messes with other stuff